packages/web-ui/src/lab/Button/Button.stories.tsx

```diff
@@ -0,0 +1,251 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Flex } from '../../Flex';
+import { Heading } from '../../Heading';
+import { Button } from './Button';
+import {
+  ChevronLeft01MediumIcon,
+  ChevronRight01MediumIcon,
+  ChevronLeft01SmallIcon,
+  ChevronRight01SmallIcon,
+  OpenMediumIcon,
+  SettingsMediumIcon,
+} from '@utilitywarehouse/react-icons';
+
+const meta: Meta<typeof Button> = {
+  title: 'Web UI / Lab / Button',
+  component: Button,
+};
+
+export default meta;
+type Story = StoryObj<typeof Button>;
+
+const sizes = ['large', 'small'] as const;
+const variants = ['solid', 'outline', 'ghost'] as const;
+const colorSchemes = {
+  solid: ['cyan', 'red', 'green'] as const,
+  outline: ['cyan', 'grey', 'red', 'green', 'gold'] as const,
+  ghost: ['cyan', 'grey', 'red', 'green', 'gold'] as const,
+};
+
+export const KitchenSink: Story = {
+  parameters: { controls: { hideNoControlsWarning: true } },
+  render: () => {
+    return (
+      <Flex direction="column" gap={6}>
```

Comment:
Nit: you can add all non-button component code as [decorators](https://github.com/utilitywarehouse/internal-web-ui/blob/main/src/components/Alert/stories/Alert.stories.meta.tsx#L11)

Reply:
aye, it's a little hard to do with the Kitchen Sink story, as there are multiple instances of _non-button component code_ but will update to decorators where possible 👍🏻